### PR TITLE
Add evidence/assessment result IDs to assessment view

### LINKED ISSUE #327: Add link to evidence in assessment view
We should add a link to the corresponding evidence or at least the evidence_id.

### PR DESCRIPTION
**Pull request by anatheka**
This PR adds an additional column to the assessment view for more information about the assessment result. For now the evidence and assessment result ID is shown. 

It also adds ordering to listCloudServiceAssessmentResults and listAssessmentResults.

Closes #327, #324

### CODE REVIEW

File: src/lib/orchestrator.ts [hunk 1]
```diff
@@ -172,7 +172,7 @@ export async function listAssessmentResults(): Promise<AssessmentResult[]> {
  * @returns an array of {@link AssessmentResult}s.
  */
 export async function listCloudServiceAssessmentResults(serviceId: string, fetch = window.fetch): Promise<AssessmentResult[]> {
-    const apiUrl = clouditorize(`/v1/orchestrator/assessment_results?pageSize=1000&filteredCloudServiceId=${serviceId}`);
+    const apiUrl = clouditorize(`/v1/orchestrator/assessment_results?pageSize=1000&filteredCloudServiceId=${serviceId}&orderBy=name&asc=true`);
```
Comment (oxisto) [inline src/lib/orchestrator.ts:175]: Why is this one sorted by name?

File: src/lib/AssessmentInfoBlock.svelte [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+	import type { AssessmentResult } from './assessment';
+	export let result: AssessmentResult;
+	let isOpen = false;
+</script>
+
+	<Button color="primary" on:click={() => (isOpen = !isOpen)}>Show more info</Button>
+	<Collapse {isOpen}>
+		<Card body class="mt-2">
+			<pre><code>Assessment Result ID: {result.id}</code></pre>
```
Comment (oxisto) [inline src/lib/AssessmentInfoBlock.svelte:12]: Code and pre seem to be redundant

File: src/lib/orchestrator.ts (hunk 1 above)
Comment (oxisto) [reply src/lib/orchestrator.ts:175]: But these are results, not cloud services? Do they even have a name?

Comment (anatheka) [reply src/lib/orchestrator.ts:175]: You're absolutely right. I only read CloudService.

Comment (anatheka) [reply src/lib/orchestrator.ts:175]: I added the sorting also to the CloudServices, but that didn't work, so I deleted it again.

**Review by oxisto — APPROVED** [review]

**Pull request merged**